src/os/bluestore/BlueStore.cc

```diff
@@ -3642,12 +3635,8 @@ void BlueStore::Onode::get() {
       ocs->lock.lock();
     }
     bool was_pinned = pinned;
-    pinned = nref >= 2;
-    // additional increment for newly pinned instance
+    pinned = nref >= 1;
```

Comment:
Looks like I'm wrong at this line - it still should be nref >= 2
Going to re-test and update the original PR.